warehouse/assemble/datawave/src/main/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ${docker.image.prefix}datawave-stack-accumulo:2.1.3
+FROM ${docker.image.prefix}datawave-stack-accumulo:${docker.image.accumulo.tag}
 USER root
 COPY --from=${docker.image.prefix}datawave-stack-hadoop:3.3.6  /usr/local/hadoop/  /usr/local/hadoop/
```

Comment:
Purely out of curiosity - should the hadoop version also be pulled out in a similar fashion or do we not care?

Reply:
Maybe, in a follow on ticket. This one is needed since were trying to update the accumulo image.